src/graphql/queries/events.ts

```diff
@@ -34,13 +42,9 @@ export const GET_ALL_EVENTS = gql`
       id
       name
       eventDate
-      ambulances {
-        id
-        vehicleNumber
-      }
-      hospitals {
+      isActive
+      createdBy {
         id
-        name
       }
```

Comment:
is this query (and these deleted fields) being used somewhere else? please double check before removing fields from shared queries

Reply:
Yeah good catch, I'm not sure why I removed this lol I'll add it back

Reply:
I did add them back, not sure why it's not displaying properly here

Reply:
can you double check your local file and make sure they're added back? as far as i can tell it's removed on this remote branch right now

Reply:
yup - for some reason, i think you're viewing an older commit. Try looking here: https://github.com/uwblueprint/paramedics-react/pull/24/files

Reply:
i'm certain i'm looking at the most up-to-date changes for this PR - even pulled it in locally and i see the same thing here
https://github.com/uwblueprint/paramedics-react/blob/42a6c89ba67d64e154404d04e6d8e79971abd53c/src/graphql/queries/events.ts#L39

regardless we can take a look at this together after standup